cfme/tests/configure/test_access_control.py

```diff
@@ -347,6 +347,8 @@ def test_delete_roles_with_assigned_group():
     group.create()
     with error.expected(flash_msg.format(role.name)):
         role.delete()
+    group.delete()
```

Comment:
Is this intended as test cleanup or as an extension of the test, to verify that the role can be deleted after the group is removed?

If its cleanup, can you place in a request.addfinalizer() call and wrap it in a try/except and warn on failure to delete?